src/Hamiltonians.jl

```diff
@@ -1371,6 +1377,24 @@ function numSandDoccupiedsites(onrep::AbstractArray)
 end
 # this one is faster by about a factor of 2 if you already have the onrep
 
+function numSandDoccupiedsites(b::BoseFS2C{NA,NB,M,AA,AB}) where {NA,NB,M,AA,AB}
```

Comment:
Is this method needed? If not, please remove!

Reply:
Removed.